src/webui/frontend/src/routes/Dashboard/Connectors.tsx

```diff
@@ -213,28 +213,61 @@ class Connectors extends React.Component<ConnectorsProps & {path: string}, OwnSt
         </Button>
       </BasicLayout>
     ) : (
-      <BasicLayout>
-        {connectors.map(c => (
-          <Card
-            key={c.name}
-            style={{width: 350, margin: 10, float: 'left'}}
-            actions={[
-              // eslint-disable-next-line
-              <a key={1} onClick={() => this.selectConnector(c.name)}>
-                <ApiOutlined /> Connect
-              </a>,
-            ]}
-          >
-            <Card.Meta
-              avatar={<Avatar src={`/icons/connectors/${c.name}.png`} />}
-              title={c.title}
-              description={c.description}
-              style={{height: 75}}
-            />
-          </Card>
-        ))}
-      </BasicLayout>
-    );
+        <BasicLayout>
+          <Tabs>
+            <Tabs.TabPane tab='Connectors' key='1'>
+              {connectors.map(c => (
+                <Card
+                  key={c.name}
+                  style={{ width: 350, margin: 10, float: 'left' }}
+                  actions={[
+                    // eslint-disable-next-line
+                    <a key={1} onClick={() => this.selectConnector(c.name)}>
+                      <ApiOutlined /> Connect
+                </a>,
+                  ]}
+                >
+                  <Card.Meta
+                    avatar={<Avatar src={`/icons/connectors/${c.name}.png`} />}
+                    title={c.title}
+                    description={c.description}
+                    style={{ height: 75 }}
+                  />
+                </Card>
+              ))}
+            </Tabs.TabPane>
+
+            <Tabs.TabPane tab='Active Connections' key='2'>
+              <Table
+                rowKey='table_name'
+                dataSource={connections.slice()}
+                pagination={false}
+                columns={[{
+                  title: 'Name',
+                  dataIndex: 'table_name',
+                  key: 'name'
+                },
+                {
+                  title: 'Created On',
+                  dataIndex: 'created_on',
+                  key: 'created_on'
```

Comment:
let's add a `render` here to format the date

```
c => (new Date(c)).toLocaleDateString()
> "10/16/2020"
```

Reply:
Good feedback.